File: compservice-annotations/src/main/java/compman/compsrv/model/events/EventType.java
package compman.compsrv.model.events;

public enum EventType {
    //Category and competition
    BRACKETS_GENERATED,
    SCHEDULE_GENERATED,
    FIGHTS_ADDED_TO_STAGE,
    STAGE_STATUS_UPDATED,


    COMPETITOR_ADDED,
    COMPETITOR_REMOVED,
    COMPETITOR_UPDATED,
    COMPETITOR_CATEGORY_CHANGED,
    COMPETITOR_CATEGORY_ADDED,

    CATEGORY_ADDED,
    CATEGORY_DELETED,
    CATEGORY_BRACKETS_DROPPED,
    CATEGORY_REGISTRATION_STATUS_CHANGED,

    COMPETITION_CREATED,
    COMPETITION_DELETED,
    COMPETITION_PROPERTIES_UPDATED,
    COMPETITORS_PROPAGATED_TO_STAGE,

    FIGHTS_START_TIME_UPDATED,
    FIGHT_PROPERTIES_UPDATED,
    FIGHTS_START_TIME_CLEANED,
    FIGHTS_EDITOR_CHANGE_APPLIED,

    SCHEDULE_DROPPED,

    REGISTRATION_PERIOD_ADDED,
    REGISTRATION_INFO_UPDATED,
    REGISTRATION_PERIOD_DELETED,
    REGISTRATION_GROUP_ADDED,
    REGISTRATION_GROUP_DELETED,
    REGISTRATION_GROUP_CATEGORIES_ASSIGNED,

    DASHBOARD_FIGHT_RESULT_SET,
    DASHBOARD_FIGHT_COMPETITORS_ASSIGNED,
    DASHBOARD_STAGE_RESULT_SET,
    MATS_UPDATED
}

File: competition-serv-model/src/main/java/compman/compsrv/model/es/events/EventType.kt
package compman.compsrv.model.es.events

enum class
EventType {
    //Category and competition
    SCORE_UPDATE,
    ABSOLUTE_CATEGORY_SAVED,
    ERROR_EVENT,
    ALL_BRACKETS_DROPPED,
    START_FIGHT,
    CHANGE_FIGHT_ORDER,
    FIGHT_RESULT_UPDATE,
    FIGHT_COMPETITORS_UPDATE,
    BRACKETS_GENERATED,
    BRACKETS_SAVED,
    SCHEDULE_GENERATED,
    SCHEDULE_SAVED,
    OVERWRITE_BRACKETS,
    CHANGE_MAT_EVENT,
    CHANGE_ORDER_EVENT,
    UPDATE_FIGHT_STAGE_EVENT,
    COMPETITOR_ADDED,

    INTERNAL_COMPETITOR_ADDED,
    INTERNAL_COMPETITOR_REMOVED,
    INTERNAL_COMPETITOR_CATEGORY_CHANGED,
    INTERNAL_ALL_BRACKETS_DROPPED,
    COMPETITOR_REMOVED,
    COMPETITOR_UPDATED,
    COMPETITOR_CATEGORY_CHANGED,
    CATEGORY_ADDED,

    CATEGORY_DELETED,
    CATEGORY_BRACKETS_DROPPED,
    CATEGORY_STATE_ADDED,
    CATEGORY_STATE_DELETED,
    COMPETITION_STARTED,

    COMPETITION_CREATED,
    COMPETITION_DELETED,
    COMPETITION_PUBLISHED,
    COMPETITION_UNPUBLISHED,
    COMPETITION_STOPPED,
    COMPETITION_PROPERTIES_UPDATED,
    FIGHTS_START_TIME_UPDATED,

    COMPETITORS_MOVED,

    SCHEDULE_DROPPED,

    //Dummy
    DUMMY,

    //Mat state
    MAT_STATE_INITIALIZED,
    DASHBOARD_STATE_INITIALIZED,
    DASHBOARD_STATE_UPDATED,
    DASHBOARD_STATE_DELETED,
    PERIOD_INITIALIZED,
    DASHBOARD_PERIOD_DELETED,
    UNDISPATCHED_MAT_ADDED,
    MAT_DELETED,
    MAT_STATE_DELETED
}